src/constants.rs

```diff
@@ -123,6 +123,12 @@ pub const DC_RESEND_USER_AVATAR_DAYS: i64 = 14;
 // do not use too small value that will annoy users checking for nonexistant updates.
 pub const DC_OUTDATED_WARNING_DAYS: i64 = 365;
 
+/// warn about a nearly full mailbox after this usage percentage is reached.
+pub const DC_QUOTA_WARN_THRESHOLD_PERCENTAGE: u64 = 90;
```

Comment:
changing this to a lower value like `10` is a great way to test it, if you don't want to fill up your mailbox for testing it.